#This message allows to store data to be shared over different agents/base station
uint8 msg_id
uint8 agent_id
hardwareInfo hardware_info
goalKeeperInfo agent_info
aiInfo ai_info
bool is_goalkeeper

